packages/terra-search-field/src/SearchField.scss

```diff
@@ -4,24 +4,26 @@
 
     .input {
       appearance: none;
+      border-bottom-left-radius: var(--terra-search-field-border-bottom-left-radius, 0.25em);
       border-bottom-right-radius: 0;
       border-right: 0;
+      border-top-left-radius: var(--terra-search-field-border-top-left-radius, 0.25em);
       border-top-right-radius: 0;
       flex: 1 1 auto;
-      margin: 0;
+      margin: var(--terra-search-field-input-margin, 0);
       min-width: 0;
       outline-offset: -0.143em;  //Adjust the outline offset for search input in Safari.
     }
 
     .button {
-      border: 0.071em solid #dedfe0;
+      border: var(--terra-search-field-button-border, 0.071em solid #dedfe0);
       border-bottom-left-radius: 0;
-      border-bottom-right-radius: 0.25em;
+      border-bottom-right-radius: var(--terra-search-field-border-bottom-right-radius, 0.25em);
       border-top-left-radius: 0;
-      border-top-right-radius: 0.25em;
+      border-top-right-radius: var(--terra-search-field-border-top-right-radius, 0.25em);
       display: inline-block;
       flex: 0 0 auto;
-      margin: 0;
+      margin: var(--terra-search-field-button-margin, 0);
```

Comment:
We've had issues using theming variables with margin/padding shorthand and bidirectional styles (LTR/RTL). Let's split this out to long-hand style.